cli/cmd/logrotate.go

```diff
@@ -47,11 +47,10 @@ func internalLogrotateModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
 	}
 
 	for _, run := range runningCtx.Instances {
-		res, err := running.Logrotate(&run)
+		err := running.Logrotate(&run)
```

Comment:
It's better to use '_, err' to ignore the unused return value.

Reply:
It doesn't return `res` anymore.

Reply:
Ooops, my mistake.